test

```diff
@@ -21,3 +21,4 @@ test coroutines-co.js
 test coroutines-bluebird.js
 test await.compiled.js
 test futures.js
+test promises2.js
```

Comment:
If we add a vanilla futures version we can rename the "Ramda" version. :)

I'm keen to avoid having __promises.js__, __promises2.js__, __promises3.js__, __futures.js__, __futures2.js__, etc. Each version should be sufficiently differentiated from each other version that it can be given a meaningfully different name.